package.json

```
@@ -21,7 +21,7 @@
   },
   "devDependencies": {
     "@nimiq/iqons": "nimiq/iqons#a3302e06b0a53b6daeb745d13dbfd930b304c791",
-    "@nimiq/style": "nimiq/nimiq-style#next",
+    "@nimiq/style": "nimiq/nimiq-style",
```

I think this can use the npm package now:
```suggestion
    "@nimiq/style": "^0.4.2",
```